modules/templates/templates_api.rb

```diff
@@ -14,8 +14,8 @@ class Proxy::TemplatesApi < Sinatra::Base
   end
 
   get "/:kind" do |kind|
-    log_halt(500, "Failed to retrieve #{kind} template for #{params[:token]}: ") do
-      Proxy::Templates::Handler.get_template(kind, params[:token], params[:static])
+    log_halt(500, "Failed to retrieve #{kind} template for #{params.inspect}: ") do
+      Proxy::Templates::Handler.get_template(kind, params, headers)
```

Comment:
Not sure what you mean, I want to verbosely copy all the headers from request to response, so `HTTP_` prefix is unwanted and the helper method `headers` is not avaible in the Handler instance.